code/__DEFINES/subsystems.dm

```diff
@@ -120,6 +120,7 @@
 #define INIT_ORDER_AI_CONTROLLERS 	55 //So the controller can get the ref
 #define INIT_ORDER_TICKER			55
 #define INIT_ORDER_MAPPING			50
+#define INIT_ORDER_TIMETRACK		47
```

Comment:
Is this used anywhere?

Reply:
![image](https://user-images.githubusercontent.com/5714543/126854999-b864c07f-5276-4621-a748-946afd08ba63.png)

Bruh